core/search/fields.py

```diff
@@ -65,10 +68,6 @@ def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
             raise FieldNotQueryableException(
                 f"Field doesn't have subfields: {'.'.join(remainder)}"
             )
-        if isinstance(expression, Range):
```

Comment:
Remove that condition from other fields as well (`ListField`, `AttributeField`, `ShareField`, `UploaderField` and `JSONField`)